interface/modules/custom_modules/oe-module-faxsms/table.sql

```diff
@@ -31,3 +31,7 @@ INSERT INTO categories(`id`,`name`, `value`, `parent`, `lft`, `rght`, `aco_spec`
 UPDATE categories SET rght = rght + 2 WHERE name = 'Categories';
 UPDATE categories_seq SET id = (select MAX(id) from categories);
 #Endif
+
+#IfMissingColumn module_faxsms_credentials updated
```

Comment:
unsure Brady. The column was missing on my dev system which has been installed and upgraded since 4.1+ but in past faxsms was a composer install.
I may need to test for both. Thanks for the catch.

Although we do need to figure out how to do upgrades w/o having to uninstall and install modules!

Doubtful we can do during normal sql upgrades. Mayne a separated patch.sql file.

Reply:
As I recall, matrix did bring in a mechanism for the modules for upgrading. Would check out this line:
https://github.com/openemr/openemr/blob/958f4ed9e455c4497d7ad4aa29596eea0005aa23/interface/modules/zend_modules/module/Installer/view/installer/installer/index.phtml#L144

Reply:
of course it's broken! Still can't see where sql_version gets set to database.

Reply:
@adunsulag 
I've fixed interface/modules/zend_modules/module/Installer/src/Installer/Controller/InstallerController.php PHP error for countable and will put up in a PR if you don't have anything to add.
I am looking into how the version gets set in module table but this looks like what we were talking about last week which is nice for us:)